Extensions/TextInput/textinputruntimeobject.ts

```diff
@@ -500,6 +535,40 @@ namespace gdjs {
     isFocused(): boolean {
       return this._renderer.isFocused();
     }
+    isSubmitted(): boolean {
+      return this._isSubmitted;
+    }
+
+    getMaxLength(): integer {
+      return this._maxLength;
+    }
+    setMaxLength(value: integer) {
+      if (this._maxLength === value) return;
+
+      this._maxLength = value;
+      this._renderer.updateMaxLength();
+    }
+    getPadding(): integer {
+      return this._padding;
+    }
+    setPadding(value: integer) {
+      if (this._padding === value) return;
```

Comment:
I think you should also prevent the function from setting negative values for the padding, there should be a minimum of 0